apps/src/makerlab/BoardController.js

```diff
@@ -3,20 +3,12 @@
  */
 /* global Promise */
 
-try {
-  // Polyfill node's process.hrtime for the browser, gets used by johnny-five.
-  process.hrtime = require('browser-process-hrtime');
-
-  var five = require('johnny-five');
-  var ChromeSerialPort = require('chrome-serialport');
-  var PlaygroundIO = require('playground-io');
-} catch (e) {
-  /**
-   * These packages should currently only be downloaded and available on
-   * /p/makerlab levels, not vanilla App Lab ones, so we ignore any require
-   * failures here.
-   */
-}
+// Polyfill node's process.hrtime for the browser, gets used by johnny-five.
+process.hrtime = require('browser-process-hrtime');
+
+import five from 'johnny-five';
```

Comment:
I think the import statement will get hoisted above the `hrtime` require here.  Is that ordering going to be a problem?  If so, you may need to leave `johnny-five` as a `require`.